package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "test": "node_modules/.bin/_mocha  $(find . -name '*.spec.js')",
-    "test:lint": "node_modules/eslint/bin/eslint.js ./  || exit 0",
+    "test:lint": "eslint .",
```

Comment:
Added new commit with passing lint now. It was just a bunch of unused stuff and `let` -> `const`. All of it was automatically fixed by eslint (after upgrading it to 3) by running `eslint . --fix`